src/index.html

```diff
@@ -11,6 +11,46 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="#"
+      method="post"
+      class="searchForm"
+      data-qa="big"
+    >
+      <label>
+        <img
+          src="./images/Search.svg"
+          alt="searchImg"
+          class="img"
+        >
+        <input
```

Comment:
Don't forget to add empty lines between multiline sibling blocks of HTML
```suggestion
        >
        
        <input
```